deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -152,15 +137,35 @@
     'tnr_macro': make_scorer(true_negative_rate_metric, averaging_method='macro'),
     'tnr_micro': make_scorer(true_negative_rate_metric, averaging_method='micro'),
     'tnr_weighted': make_scorer(true_negative_rate_metric, averaging_method='weighted'),
+    'jaccard_macro': make_scorer(jaccard_score, average='macro', zero_division=0),
+    'jaccard_micro': make_scorer(jaccard_score, average='micro', zero_division=0),
+    'jaccard_weighted': make_scorer(jaccard_score, average='weighted', zero_division=0),
+    'jaccard_per_class': make_scorer(jaccard_score, average=None, zero_division=0),
+}
+
+binary_scorers_dict = {
+    'balanced_accuracy': get_scorer('balanced_accuracy'),
+    'average_precision': get_scorer('average_precision'),
```

Comment:
average precision is not precision marco?

Reply:
No it's the average over the precision recall curve if I remember correctly